apps/anomalist/gp_detector.py

```diff
@@ -118,8 +118,8 @@ def get_score_df(self, df: pd.DataFrame, variable_ids: List[int], variable_mappi
             # Get the data for the current entity and variable
             group = df_wide.loc[(entity_name, variable_id)]
 
-            # Skip if the series has only one or fewer data points
-            if isinstance(group, pd.Series) or len(group) <= 1:
+            # Skip if the series has only three or fewer data points
+            if isinstance(group, pd.Series) or len(group) <= 3:
```

Comment:
Good idea, maybe this should be applied to all detectors, not just GP (up to you).

Reply:
I'll leave this decision up to you. GP is not useful with just 3 points, but other detectors might be.